implementations/micrometer-registry-statsd/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'com.gradleup.shadow' version '8.3.6'
+    id 'com.gradleup.shadow' version '8.3.9'
```

Comment:
Should we create an issue to migrate to 9.x of the plugin (I guess it won't be straightforward because of lots of breaking changes)?

Reply:
Created: https://github.com/micrometer-metrics/micrometer/issues/6938